### ISSUE TITLE
Paste option list requires admin??

### ISSUE THREAD

**Issue by EngelPika32** [nextcloud/polls#2560]
tl;dr: Check the last line.
- - - - -

## What is going wrong? What did you observe?
**Describe the bug**

Receiving 403 with the message "Logged in user must be an admin" as a non-admin user when adding options in bulk.

**Steps to reproduce the behavior**
1. Create a Text-Poll
2. Select "Paste option list" from the Options
3. Insert some options
4. Press "OK"
5. A small "Error" Box appears in the top-right corner and fades away after a few seconds
6. Open the Browser-Console and observe the described 403 request response 
7. Do steps 1 to 3 as an admin user
8. All options are added correctly, no errors; status code: 200

**Add a screenshot of the misbehavior**

> If applicable, add screenshots to help explain your problem.

Do you need an image pointing to the options, or? … 

## How should it work?
**What you expected to happen?**

Normal users should be able to quickly add multiple text answer options just like admins can.

## Information about your polls installation
**Polls version:**  3.7.0

**How did you install this version?(Appstore or describe installation)**

- [x] Installed from the appstore
- [ ] Installed via occ
- [ ] Installed via extracting downloaded package to the apps folder

## Information about your Instance of Nextcloud
**Nextcloud version:** 23.0.8

### Server configuration (decide, if you think it is helpful)

**PHP version:**
- [ ] 7.3 or lower
- [ ] 7.4
- [x] 8.0

 

## Logs

<details>
<summary>Nextcloud log (data/nextcloud.log)</summary>

No error/warning logs

</details>

<details>
<summary>Browser Console Log</summary>

```
→ POST https://mydomain.tld/apps/polls/option/bulk 403
→ Error adding option: [object Object] {error: {…}}
error: {
	config: {transitional: {…}, transformRequest: Array(1), transformResponse: Array(1), timeout: 0, adapter: ƒ, …}
	data: {message: 'Logged in user must be an admin'}
	headers: {…}
	request: XMLHttpRequest {onreadystatechange: null, readyState: 4, timeout: 0, withCredentials: false, upload: XMLHttpRequestUpload, …}
	status: 403
	statusText: ""
}
{payload: {…}} payload: {text: 'op1\nop2\nop3'}
``` 
*(actual payload replaced by sample values)*

</details>

- - - - - -

I noticed https://github.com/nextcloud/polls/blob/cbb8defb9677fe962ca7ac83a84f450fe0774506/lib/Controller/OptionController.php#L69 is the only one in this file without the `@NoAdminRequired` annotation. So, this is probably a one-line fix?

**Comment by dartcafe** [comment]
Hmmm, nice find. Yes, the annotation is missing. You could push it straight into a PR.
